python/app.yaml

```diff
@@ -1,7 +1,7 @@
 runtime: python37
 service: python
 
-instance_class: F4
+instance_class: F4_1G
```

Comment:
add a comment here specifying why it is needed

Reply:
With this new lemmatizer model, we are now consuming ~1.2 GB of memory per request, which can only be served by a F4_1G instance.